src/schema/ecommerce/types/offer.ts

```diff
@@ -67,6 +72,11 @@ export const OfferType = new GraphQLObjectType({
       type: OrderInterface,
       description: "The order on which the offer was made",
     },
+    buyerTotalCents: {
+      type: GraphQLInt,
+      description: "Offer amount in cents",
+    },
+    buyerTotal: amount(({ amountCents }) => amountCents),
```

Comment:
didn't get a chance to review this before it was merged, but shouldn't  this be 

```
buyerTotal: amount(({ buyerTotalCents }) => buyerTotalCents),
```

?

Reply:
oh good catch! i'll update it now! sorry